src/MassTransit/SqlTransport/Configuration/SqlTransportOptions.cs

```diff
@@ -23,4 +23,28 @@ public class SqlTransportOptions
     /// If specified, changes the connection limit from the default value (10)
     /// </summary>
     public int? ConnectionLimit { get; set; }
+
+    public AuthenticationMode AuthenticationMode { get; set; } = AuthenticationMode.Password;
+
+    public TokenIssuerOptions? TokenIssuerOptions { get; set; }
+}
+
+
+public class TokenIssuerOptions
```

Comment:
This would be unnecessary if the credentials were pulled from the container provider vs. configuration.